osu.Game.Rulesets.Catch/UI/Catcher.cs

```diff
@@ -312,6 +309,17 @@ public void SetHyperDashState(double modifier = 1, float targetPosition = -1)
             }
         }
 
+        /// <summary>
+        /// Set the scale and catch width.
+        /// </summary>
+        public void SetScaleAndCatchWidth(IBeatmapDifficultyInfo? difficulty)
```

Comment:
Would prefer renaming to something like `ApplyDifficulty(IBeatmapDifficultyInfo?)` or something.